docs/integrations/containers-orchestration/vmware.md

```diff
@@ -74,22 +75,22 @@ This section walks you through the process of installing Sumo Logic scripts for
 
 To install and configure the Sumo Logic scripts, do the following:
 
-1. On the server, host, or VM create a directory in which to put the Sumo Logic scripts from [Sumo Logic Scripts](https://github.com/SumoLogic/sumologic-vmware/tree/master/vsphere) for VMware. We recommend that you name the directory **/var/log/vmware**, or something similar.
+1. On the server, host, or VM create a directory in which to put the Sumo Logic scripts from [Sumo Logic Scripts](https://github.com/SumoLogic/sumologic-vmware/tree/master/vsphere) for VMware. We recommend that you name the directory **/var/log/vmware* or something similar.
 2. Download the Sumo Logic VMware scripts from [here](https://s3.amazonaws.com/appdevstore/VMWare/sumo-vsphere-ulm.zip), into the directory you just created.
 3. Install [python](https://www.python.org/) version 3.6, or later.
 4. Install [pyvmomi](https://pypi.org/project/pyvmomi/) 6.7.3:
-  ```bash
-  pip install pyvmomi==6.7.3
-  ```
+   ```bash
+   pip install pyvmomi==6.7.3
```

Comment:
@ankurch627 - We can point the user to this doc : https://github.com/vmware/pyvmomi?tab=readme-ov-file#compatibility-policy for choosing the version of this library based on there vsphere version